### ISSUE TITLE
[SUGGESTION] Change color of italics

### ISSUE THREAD

**Issue by firinael** [hydescarf/Obsidian-Theme-Mado-Miniflow#12]
Italics are meant to be a type of emphasis, yet they're basically greyed out in this theme. I heavily recommend changing their color to at least the default text color, or even coloring them in something that actually calls attention.

**Comment by hydescarf** [comment]
Make senses actually, I'll look into this!

**Comment by hydescarf** [comment]
Colours are adjusted in the latest update! https://github.com/hydescarf/Obsidian-Theme-Mado-Miniflow/commit/d7746671473e8c10908bd09cd6888a5d307c360c
I've switched them from gray to another shade of blue, hopefully it doesn't stand out much. I personally think that italic shouldn't be too obvious, as its italic style is already very hard to detect and to read, gray colour would distinguish it from the normal text easily and remain something as "hard to read", thus the choice previously. But yeah, it is now a shade of blue.

I've also adjusted the colour for Bolding as well, since it's hardly distinguishable with the previous dark-grayish colour from normal text.
Let me know if you think the chosen colour is still up to debate for Italic.
